### ISSUE TITLE
Add commenting functionality

### ISSUE THREAD

**Issue by manmal** [manmal/hn-android#37]

**Comment by Chris911** [comment]
Is there an ETA for this feature?

**Comment by peshkira** [comment]
No, unfortunately not. This is a side project for us and we try to allocate time for it as much as possible, but we cannot give estimations. If you would like and have some time to contribute, I'll be glad to give you a few pointers, if needed.

**Comment by Chris911** [comment]
For sure. Currently on vacation but if you have time to do a little write-up on how this feature should be implemented I might find the time to do it. It's time consuming to jump into a new project but with a little "spec" it shouldn't be too hard. If you have time for the let me know (email, here).

**Comment by peshkira** [comment]
Nice, I'll do that asap.
